app/views/users/new.html.erb

```diff
@@ -0,0 +1,45 @@
+<!--<form accept-charset="UTF-8" action="/users" method="post">
```

Comment:
- [ ] uses users_path helper for action

- change action="/users" to action=users_path

Reply:
Change implemented! Thanks for the commentary.
Happy coding